src/middlewared/middlewared/plugins/pool.py

```diff
@@ -1942,6 +1948,13 @@ class Config:
         namespace = 'pool.dataset'
         event_send = False
 
+    @accepts()
+    async def compression_algorithm_choices(self):
```

Comment:
Should probably be just `compression_choices` to map to the attribute name like we do with all other choices